Command/SshCommand.php

```diff
@@ -84,6 +90,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $username = $input->getArgument('username');
         $service = $input->getOption('service');
         $bindAddress = $input->getOption('bind-address');
+        $toolsDb = $input->hasOption('toolsdb');
         $host = "$service".self::HOST_SUFFIX;
         $login = $username.'@'.self::LOGIN_URL;
 
```

Comment:
I think the output line below here should say something if it's also tunneling toolsdb, otherwise there's no difference in output with `--toolsdb` and Sams get confused.